frontends/mit-open/src/pages/HomePage/TestimonialsSection.tsx

```diff
@@ -0,0 +1,254 @@
+import React from "react"
+import {
+  Container,
+  Typography,
+  styled,
+  theme,
+  pxToRem,
+  ActionButton,
+  TruncateText,
+} from "ol-components"
+import { useTestimonialList } from "api/hooks/testimonials"
+import { RiArrowDropRightLine, RiArrowDropLeftLine } from "@remixicon/react"
+import Slider from "react-slick"
+
+const Section = styled.section(({ theme }) => ({
+  backgroundColor: theme.custom.colors.mitRed,
+  color: theme.custom.colors.white,
+  overflow: "auto",
+  padding: "80px 0",
+  [theme.breakpoints.down("md")]: {
+    padding: "40px 0",
+  },
+  "h2, h3": {
+    textAlign: "center",
+  },
+  h3: {
+    marginTop: "8px",
+    marginBottom: "60px",
+    ...theme.typography.body1,
+  },
+}))
+
+const TestimonialCardContainer = styled.div({
+  width: "100%",
+  [theme.breakpoints.down("md")]: {
+    padding: "0 10%",
+  },
+})
+
+const TestimonialCard = styled.div({
+  height: "326px",
+  backgroundColor: theme.custom.colors.white,
+  color: theme.custom.colors.black,
+  display: "flex",
+  borderRadius: "8px",
+  margin: "0 0 50px 24px",
+  [theme.breakpoints.down("md")]: {
+    height: "411px",
+    flexDirection: "column",
+    margin: "0",
+  },
+})
+
+const TestimonialCardImage = styled.div({
+  height: "326px",
+  img: {
+    height: "326px",
+    objectFit: "cover",
+    borderTopLeftRadius: "8px",
+    borderBottomLeftRadius: "8px",
+    [theme.breakpoints.down("md")]: {
+      width: "100%",
+      height: "190px",
+      borderTopRightRadius: "8px",
+      borderBottomLeftRadius: "0px",
+    },
+  },
+  [theme.breakpoints.down("md")]: {
+    width: "100%",
+    height: "190px",
+    borderRadius: "8px 0 8px 0",
+    backgroundColor: theme.custom.colors.darkBlue,
+  },
+})
+
+const TestimonialCardQuote = styled.div({
+  height: "326px",
+  backgroundColor: theme.custom.colors.white,
+  color: theme.custom.colors.black,
+  padding: "0 32px 32px",
+  flexDirection: "column",
+  justifyContent: "space-between",
+  alignItems: "flex-start",
+  flex: "1 0 0",
+  alignSelf: "stretch",
+  borderRadius: "8px",
+  display: "flex",
+  [theme.breakpoints.down("md")]: {
+    width: "100%",
+    height: "161px",
+    borderRadius: "0 8px 0 8px",
+    marginTop: "16px",
+    marginBottom: "16px",
+    padding: "0 16px",
+  },
+
+  "div.testimonial-quote-opener": {
+    color: theme.custom.colors.mitRed,
+    fontStyle: "normal",
+    height: pxToRem(80),
+    width: "100%",
+    fontSize: pxToRem(80),
+    lineHeight: pxToRem(120),
+    [theme.breakpoints.down("md")]: {
+      fontSize: pxToRem(80),
+      height: pxToRem(50),
+      lineHeight: "normal",
+    },
+  },
+
+  h4: {
+    flexGrow: "1",
+    [theme.breakpoints.down("md")]: {
+      width: "100%",
+      ...theme.typography.subtitle1,
+    },
+  },
+
+  "div.testimonial-quote-closer": {
+    textAlign: "right",
+    width: "100%",
+  },
+})
+
+const OverlayContainer = styled.div({
+  position: "relative",
```

Comment:
```suggestion
  position: "relative",
  maxWidth: "1900px",
  margin: "0 auto",
```
Per this discussion https://mitodl.slack.com/archives/C03K5HYGPT9/p1717533212840589?thread_ts=1717523918.589909&cid=C03K5HYGPT9, let's add a max width beyond our usual 1272px.